modules/oxide/gulpfile.js

```diff
@@ -13,7 +13,7 @@ const fs = require('fs');
 const path = require('path');
 
 const autoprefix = new lessAutoprefix({
-  browsers: ['IE 11', 'last 2 Safari versions', 'iOS 9.0', 'last 2 Chrome versions', 'Firefox ESR'],
+  browsers: ['last 2 Safari versions', 'iOS 13.0', 'last 2 Chrome versions', 'Firefox ESR'],
```

Comment:
Should this be `iOS 14.0` for 6.0? I just went with the current documented values but given we only support the last 2 majors we should be changing that for 6.0 IIUC.

Reply:
yes, 15 is current

Reply:
Cheers, have changed this to iOS 14 at minimum now then